packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -1936,42 +1951,46 @@ export const accountRoutes = (
-            authPWVersion2: validators.authPWVersion2
-            .optional()
-            .description(DESCRIPTION.authPWVersion2),
-            wrapKb: validators.wrapKb
-              .optional()
-              .description(DESCRIPTION.wrapKb),
-            wrapKbVersion2: validators.wrapKb
-              .optional()
-              .description(DESCRIPTION.wrapKbVersion2),
-            clientSalt: validators.clientSalt
-              .optional()
-              .description(DESCRIPTION.clientSalt),
-            service: validators.service.description(DESCRIPTION.service),
-            redirectTo: validators
-              .redirectTo(config.smtp.redirectDomain)
-              .optional()
-              .description(DESCRIPTION.redirectTo),
-            resume: isA
-              .string()
-              .max(2048)
-              .optional()
-              .description(DESCRIPTION.resume),
-            metricsContext: METRICS_CONTEXT_SCHEMA,
-            style: isA.string().allow('trailhead').optional(),
-            verificationMethod: validators.verificationMethod
-              .optional()
-              .description(DESCRIPTION.verificationMethod),
-            // preVerified is not available in production mode.
-            ...(!(config as any).isProduction && {
-              preVerified: isA.boolean(),
-            }),
-          })
-          .and('authPWVersion2', 'wrapKb', 'wrapKbVersion2', 'clientSalt'),
+          payload: isA
+            .object({
+              email: validators
+                .email()
+                .required()
+                .description(DESCRIPTION.email),
+              authPW: validators.authPW.description(DESCRIPTION.authPW),
+              authPWVersion2: validators.authPWVersion2
+                .optional()
+                .description(DESCRIPTION.authPWVersion2),
+              wrapKb: validators.wrapKb
+                .optional()
+                .description(DESCRIPTION.wrapKb),
+              wrapKbVersion2: validators.wrapKb
+                .optional()
+                .description(DESCRIPTION.wrapKbVersion2),
+              clientSalt: validators.clientSalt
+                .optional()
+                .description(DESCRIPTION.clientSalt),
+              service: validators.service.description(DESCRIPTION.service),
+              redirectTo: validators
+                .redirectTo(config.smtp.redirectDomain)
+                .optional()
+                .description(DESCRIPTION.redirectTo),
+              resume: isA
+                .string()
+                .max(2048)
+                .optional()
+                .description(DESCRIPTION.resume),
+              metricsContext: METRICS_CONTEXT_SCHEMA,
+              style: isA.string().allow('trailhead').optional(),
+              verificationMethod: validators.verificationMethod
+                .optional()
+                .description(DESCRIPTION.verificationMethod),
+              // preVerified is not available in production mode.
```

Comment:
And perhaps not available in local dev mode either 🤔

Reply:
This is from Dan's changes.  They might disappear if https://github.com/mozilla/fxa/pull/16240 gets merged first.

Anyways, I've seen preVerified used in tests.  Hard to believe that's no longer the case.